Core/Types/CkanModule.cs

```diff
@@ -194,6 +201,9 @@ public class CkanModule : IEquatable<CkanModule>
         [JsonProperty("install")]
         public ModuleInstallDescriptor[] install;
 
+        // Used to see if we're compatible with a given game/KSP version or not.
+        private IGameComparator _comparator;
```

Comment:
If `_comparator` is only ever set in a constructor (which I think it is) it can also be made `readonly`.